risc0/zkp/rust/src/core/sha.rs

```diff
@@ -35,6 +35,11 @@ pub const DIGEST_WORDS: usize = 8;
 /// The size of a word within a [Digest] (32-bits = 4 bytes).
 pub const DIGEST_WORD_SIZE: usize = mem::size_of::<u32>();
 
+/// Standard SHA initialization vector .
+pub static SHA256_INIT: Digest = Digest::new([
+    0x6a09e667, 0xbb67ae85, 0x3c6ef372, 0xa54ff53a, 0x510e527f, 0x9b05688c, 0x1f83d9ab, 0x5be0cd19,
+]);
```

Comment:
I think this is already defined in 2 places in rust already. We can have a followup PR to consolidate.